sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -141,6 +142,12 @@ private[sql] trait SQLConf {
     getConf(PARQUET_BINARY_AS_STRING, "false").toBoolean
 
   /**
+   * When set to true, we always treat INT96Values in Parquet files as timestamp.
+   */
+  private[spark] def isParquetINT96AsTimestamp: Boolean =
+    getConf(PARQUET_INT96_AS_TIMESTAMP, "false").toBoolean
```

Comment:
We don't really use INT96 for anything else (and I don't think other systems do either?) so maybe this should be true by default?